Jenkinsfile

```diff
@@ -306,3 +311,18 @@ timestamps {
     }
 }
 
+if (env.CHANGE_ID) {
+    // This is a pull request build
+    node('inttest') {
+        stage('Integration Test') {
+            checkout scm
+            unstash 'event-formation-unit-centos7.tar.gz'
+            sh "tar xzvf event-formation-unit-centos7.tar.gz"
+            sh "ls -R"
+            sh """
+                export LD_LIBRARY_PATH=\$LD_LIBRARY_PATH:./event-formation-unit/lib/
+                python3 -u ./utils/efushell/integrationtest.py
```

Comment:
The integration test is well hidden here. It is also less a utility and more athing in its own right. Maybe put this in its own folder at the top level so it is easy to spot (or in its ownfolder below utils).

Also consider updating README.md